### ISSUE TITLE
Lower mAP obtained when training on RGBNT201

### ISSUE THREAD

**Issue by zw-absin** [924973292/DeMo#6]
Hello,

Thank you for the code. I trained the model on RGBNT201 using an A6000 GPU, and my results are fluctuating around 75%. I kept all the original configuration parameters unchanged.

Could this discrepancy in performance (compared to the expected 79%) be due to hardware differences? 

**Best result with ViT-B**:
![Image](https://github.com/user-attachments/assets/77f5e8d0-6964-4967-b79c-8b171fe16836)
 **Result with vit_base_patch16_224**
![Image](https://github.com/user-attachments/assets/4f7e11f4-ef32-4188-975e-d62212a53073)

Here are the specific details of my setup:
**GPU:** A6000
**Framework:** 
![Image](https://github.com/user-attachments/assets/dba29c58-696e-477b-8033-320f11f79a71)
**Config:**
MODEL:
  TRANSFORMER_TYPE: 'ViT-B-16'
  STRIDE_SIZE: [ 16, 16 ]
  SIE_CAMERA: True
  DIRECT: 1
  SIE_COE: 1.0
  ID_LOSS_WEIGHT: 0.25
  TRIPLET_LOSS_WEIGHT: 1.0
  GLOBAL_LOCAL: True
  HDM: True
  ATM: True
  HEAD: 4

INPUT:
  SIZE_TRAIN: [ 256, 128 ]
  SIZE_TEST: [ 256, 128 ]
  PROB: 0.5 # random horizontal flip
  RE_PROB: 0.5 # random erasing
  PADDING: 10

DATALOADER:
  SAMPLER: 'softmax_triplet'
  NUM_INSTANCE: 8
  NUM_WORKERS: 14

DATASETS:
  NAMES: ('RGBNT201')
  ROOT_DIR: '/data/Dataset/'

SOLVER:
  BASE_LR: 0.00035
  WARMUP_ITERS: 10
  MAX_EPOCHS: 50
  OPTIMIZER_NAME: 'Adam'
  IMS_PER_BATCH: 64
  EVAL_PERIOD: 1

TEST:
  IMS_PER_BATCH: 128
  RE_RANKING: 'no'
  WEIGHT: ''
  NECK_FEAT: 'before'
  FEAT_NORM: 'yes'
  MISS: "nothing"

OUTPUT_DIR: './DeMo_201'

Any advice would be greatly appreciated.

**Comment by 924973292** [comment]
Hi, what Python version are you using? 
Should it be conda create -n DeMo python=3.10.13?

**Comment by 924973292** [comment]
Your sample speed seems abnormal. Additionally, hardware can indeed make a difference of around 2 points in mAP. I’ve tested it on A800/A100, and it can reach 79 mAP or even higher.

**Comment by zw-absin** [comment]
Thank you for your response. I am using Python 3.10.16.

**Comment by 924973292** [comment]
I found the initial environment configuration for this article, which will be published in this response today, and the ReadMe will be updated accordingly. Thank you for your attention.

**Comment by zw-absin** [comment]
Thanks a lot! and thank you for your contributions in the field.

**Comment by 924973292** [comment]
```bash
conda create -n DeMo python=3.8.12 -y 
conda activate DeMo
pip install torch==1.13.1+cu117 torchvision==0.14.1+cu117 torchaudio==0.13.1+cu117 --extra-index-url https://download.pytorch.org/whl/cu117
cd (your_path)
pip install -r requirements.txt
python train_net.py --config_file configs/RGBNT201/DeMo.yml
```

Please try the above environment configuration. 

I have synchronized the changes in README, and below is the result I just obtained on one A800 GPU:

<img width="737" alt="Image" src="https://github.com/user-attachments/assets/b3647590-3277-41b3-8d55-86350feb0f5b" />

The training process and verifiable weights can be found at this link:
[BAIDU](https://pan.baidu.com/s/1792Jwo31J7cVj8XtCTkGOg ) -- Code : olh1

Thank you for your attention! 

Our latest work, [IDEA](https://github.com/924973292/IDEA), has also been open-sourced, and you can try it out as well! 

🌟🌟🌟🌟
